mergetools/kdiff3

```diff
@@ -28,4 +28,4 @@ exit_code_trustable () {
 
 translate_merge_tool_path() {
     which kdiff3 || mergetool_find_win32_cmd "kdiff3.exe" "Kdiff3"
-}
\ No newline at end of file
+}
```

Comment:
You will want to squash this into the previous patch that needed to be fixed up. Learn how to do that e.g. [here](https://blog.axosoft.com/learning-git-interactive-rebase/), using an _interactive rebase_.